WaarpR66/src/main/java/org/waarp/openr66/client/utils/OutputFormat.java

```diff
@@ -286,8 +327,11 @@ public String toString(OUTPUTFORMAT format) {
               });
           final StringBuilder builder = new StringBuilder("<xml>");
           for (final Entry<String, Object> entry : map.entrySet()) {
+            String value = entry.getValue().toString();
+            value = value.replace('<', '[');
+            value = value.replace('>', ']');
```

Comment:
I think it is best when possible, not to change the values.
In XML, two chars must not be in the content: `<` and `&`.

This is more resilient:
```java
value = value.replace("&", "&amp;").replace("<", "&lt;");
```

Reply:
I understand, but it seems very ugly for end users... Worst than replacing unalowed characters (need to undecode while not really undecodable since simple String).
So I would propose to replace `<` and `>` by still `[` and `]` and replace `&` by `:` (or something else if better)
Still parsable...

Reply:
See proposal and tests